api/python/arrus/devices/us4r.py

```diff
@@ -304,20 +307,30 @@ def get_dto(self):
 
     def get_data_description(self, upload_result, sequence):
         # Prepare data buffer and constant context metadata
+        fcm = self._get_fcm(upload_result, sequence)
+        return arrus.metadata.EchoDataDescription(
+            sampling_frequency=self.current_sampling_frequency,
+            custom={"frame_channel_mapping": fcm}
+        )
+
+    def get_data_description_updated_for_subsequence(self, upload_result, sequence):
+        fcm = self._get_fcm(upload_result, sequence)
+        return arrus.metadata.EchoDataDescription(
+            sampling_frequency=self.current_sampling_frequency,
+            custom={"frame_channel_mapping": fcm}
+        )
+
+    def _get_fcm(self, upload_result, sequence):
```

Comment:
Should be some description/docstring for this method.

Reply:
Done.